x-pack/test_serverless/api_integration/test_suites/common/core/ui_settings.ts

```diff
@@ -18,34 +18,32 @@ const DEFAULT_COLUMNS_SETTING = 'defaultColumns';
 const TEST_SETTING = 'testSetting';
 
 export default function ({ getService }: FtrProviderContext) {
-  const svlCommonApi = getService('svlCommonApi');
-  const svlUserManager = getService('svlUserManager');
-  let roleAuthc: RoleCredentials;
-  const supertestWithoutAuth = getService('supertestWithoutAuth');
+  const roleScopedSupertest = getService('roleScopedSupertest');
+  let supertestAdminWithCookieCredentials: SupertestWithRoleScopeType;
 
   describe('ui settings service', () => {
     before(async () => {
-      roleAuthc = await svlUserManager.createM2mApiKeyWithRoleScope('admin');
+      supertestAdminWithCookieCredentials = await roleScopedSupertest.getSupertestWithRoleScope(
+        'admin',
+        {
+          useCookieHeader: true,
+          withInternalHeaders: true,
+        }
+      );
```

Comment:
Since all the APIs in this suite are `internal`, we use supertest scoped to `Admin` role and cookie header for auth